app/containers/wallet/staking/StakingDashboardPage.js

```diff
@@ -1062,19 +1054,16 @@ export default class StakingDashboardPage extends Component<Props> {
     }
     const api = getApiForNetwork(selected.getParent().getNetworkInfo());
     const time = (() => {
-      if (api === ApiOptions.ada) {
-        return {
-          getTimeCalcRequests: stores.substores.ada.time.getTimeCalcRequests,
-          getCurrentTimeRequests: stores.substores.ada.time.getCurrentTimeRequests,
-        };
-      }
       if (api === ApiOptions.jormungandr) {
         return {
           getTimeCalcRequests: stores.substores.jormungandr.time.getTimeCalcRequests,
           getCurrentTimeRequests: stores.substores.jormungandr.time.getCurrentTimeRequests,
         };
       }
-      throw new Error(`${nameof(EpochProgressContainer)} api not supported`);
+      return {
```

Comment:
This definitely doesn't look right. Why would you return timestores for ADA for example when you have Ergo selected? I get that this page should never be loaded when Ergo is used, but still this feels like a hacky solution. Maybe it makes more sense to return undefined here and then in `componentDidMount` do nothing if the result is undefined

Reply:
fixed as we discussed